src/main/fc/stats.c

```diff
@@ -24,13 +24,15 @@ PG_RESET_TEMPLATE(statsConfig_t, statsConfig,
     .stats_enabled = SETTING_STATS_DEFAULT,
     .stats_total_time = SETTING_STATS_TOTAL_TIME_DEFAULT,
     .stats_total_dist = SETTING_STATS_TOTAL_DIST_DEFAULT,
+    .stats_flight_count = SETTING_STATS_FLIGHT_COUNT_DEFAULT,
 #ifdef USE_ADC
     .stats_total_energy = SETTING_STATS_TOTAL_ENERGY_DEFAULT
 #endif
 );
 
 static uint32_t arm_millis;
 static uint32_t arm_distance_cm;
+static uint32_t prev_flight_count;
```

Comment:
I don't think this variable is needed. You can just use the value in statsConfig and statsConfigMutable.

Reply:
Resolved, but please update to uint16_t 👍🏻